src/main/java/lotto/model/LottoRank.java

```diff
@@ -0,0 +1,73 @@
+package lotto.model;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum LottoRank {
+	FIRST(6, 0, 2_000_000_000),
+	SECOND(5, 1, 30_000_000),
+	THIRD(5, 0, 1_500_000),
+	FORTH(4, 0, 50_000),
+	FIFTH(3, 0, 5_000),
+	NOTHING(0, 0, 0);
+
+	private int containsCount;
+	private int bonusCount;
+	private int money;
+
+	LottoRank(int containsCount, int bonusCount, int money) {
+		this.containsCount = containsCount;
+		this.bonusCount = bonusCount;
+		this.money = money;
+	}
```

Comment:
```suggestion
	FIRST(6, 0, new Money(2_000_000_000),
	SECOND(5, 1, new Money(30_000_000),
	THIRD(5, 0, new Money(1_500_000),
	FORTH(4, 0, new Money(50_000),
	FIFTH(3, 0, new Money(5_000),
	NOTHING(0, 0, new Money(0);

	private int containsCount;
	private int bonusCount;
	private Money money;

	LottoRank(int containsCount, int bonusCount, Money money) {
		this.containsCount = containsCount;
		this.bonusCount = bonusCount;
		this.money = money;
	}
```

Money 클래스를 도출하셨으니 다음과 같이 원시 값을 포장해보는건 어떨까요?

Reply:
👍 👍